### DISCUSSION TITLE
Un-handled Error in Documents microservices

### DISCUSSION THREAD

**Discussion by baarsamo** [opencrvs/opencrvs-core#7887] [Support requests]
Hi this is Mohamed from Somalia, we are live in production and we got this error in our slack alerts. I raised this bug please investigate as soon as possible and let me know if this is critical or not. please contact me if you need any more information. Many thanks
https://github.com/opencrvs/opencrvs-core/issues/7886

**Comment by naftis** [comment]
Closing as duplicate of the issue
